engines/kinduction.h

```diff
@@ -23,10 +23,10 @@ namespace pono {
 class KInduction : public Prover
 {
  public:
-  KInduction(const Property & p, smt::SmtSolver & solver);
-  KInduction(const PonoOptions & opt,
-             const Property & p,
-             smt::SmtSolver & solver);
+  KInduction(const Property & p, smt::SolverEnum se);
+  KInduction(const Property & p, smt::SmtSolver solver);
```

Comment:
const ref to SmtSolver

Reply:
Thanks, addressed with 0dbc624d1d5d9ab57c7977b758211620fdff5ecd